### PR TITLE
GEOMESA-1431 Enable Hbase, Bigtable, Kafka09, Kafka10 profiles

### PR DESCRIPTION
**Pull request by jahhulbert-ccri**
Note: Stacks on top of #1094 only consider last commit

### CODE REVIEW

File: build/cqs.tsv [hunk 1]
```diff
@@ -1,5 +1,5 @@
+asm:asm	3.1	compile
```
Comment (jahhulbert-ccri) [inline build/cqs.tsv]: https://dev.eclipse.org/ipzilla/show_bug.cgi?id=10583

**Comment by jahhulbert-ccri** [comment]
rebased and force pushed...retesting build now

File: geomesa-gs-plugin/geomesa-kafka-gs-plugin/pom.xml [hunk 2]
```diff
@@ -25,8 +25,10 @@
 
     <modules>
         <module>geomesa-kafka-08-gs-plugin</module>
-        <module>geomesa-kafka-09-gs-plugin</module>
-        <module>geomesa-kafka-10-gs-plugin</module>
+        
+        <!-- these are disabled and built with profiles -Pkafka09 and -Pkafka10 pending CQs -->
+        <!--<module>geomesa-kafka-09-gs-plugin</module>-->
+        <!--<module>geomesa-kafka-10-gs-plugin</module>-->
```
Comment (elahrvivaz) [inline geomesa-gs-plugin/geomesa-kafka-gs-plugin/pom.xml]: don't you need the profile in this file or not?

File: pom.xml [hunk 3]
```diff
@@ -322,6 +322,28 @@
             </pluginRepositories>
         </profile>
         <profile>
+            <id>kafka09</id>
+            <modules>
+                <module>geomesa-kafka/geomesa-kafka-datastore/geomesa-kafka-09-datastore</module>
```
Comment (elahrvivaz) [inline pom.xml]: ah, didn't realize you could do nested paths in profiles

File: build/cqs.tsv (hunk 1 above)
Comment (elahrvivaz) [inline build/cqs.tsv]: already approved, nice

**Pull request merged**